lib/metasploit/framework/mssql/client.rb

```diff
@@ -1,15 +1,65 @@
 require 'metasploit/framework/tcp/client'
 require 'metasploit/framework/mssql/tdssslproxy'
 require 'metasploit/framework/mssql/base'
+require 'msf/core/exploit'
+require 'msf/core/exploit/remote'
 
 module Metasploit
   module Framework
     module MSSQL
 
       module Client
         extend ActiveSupport::Concern
+
         include Metasploit::Framework::Tcp::Client
         include Metasploit::Framework::MSSQL::Base
+        include Msf::Exploit::Remote::MSSQL_COMMANDS
+        include Msf::Exploit::Remote::Udp
+        include Msf::Exploit::Remote::NTLM::Client
+        include Msf::Exploit::Remote::Kerberos::Ticket::Storage
+        include Msf::Exploit::Remote::Kerberos::ServiceAuthenticator::Options
+
+        attr_accessor :tdsencryption
+        attr_accessor :sock
+        attr_accessor :auth
+        attr_accessor :ssl
+        attr_accessor :ssl_version
+        attr_accessor :ssl_verify_mode
+        attr_accessor :ssl_cipher
+        attr_accessor :proxies
+        attr_accessor :connection_timeout
+        attr_accessor :send_lm
+        attr_accessor :send_ntlm
+        attr_accessor :send_spn
+        attr_accessor :use_lmkey
+        attr_accessor :use_ntlm2_session
+        attr_accessor :use_ntlmv2
+        attr_accessor :windows_authentication
+        
+        # @!attribute max_send_size
+        #   @return [Integer] The max size of the data to encapsulate in a single packet
+        attr_accessor :max_send_size
+        # @!attribute send_delay
+        #   @return [Integer] The delay between sending packets
+        attr_accessor :send_delay
+
+        included do
```

Comment:
I'm guessing when we convert this to a class, we'll need to make sure we drop this `included` callback here - as we will want the validation to be present